lib/stellar_core_commander/docker_process.rb

```diff
@@ -160,11 +162,16 @@ def dump_cores
 
     Contract None => Any
     def dump_database
-      fname = "#{working_dir}/database-#{Time.now.to_i}-#{rand 100000}.sql"
-      $stderr.puts "dumping database to #{fname}"
-      res = @state_container.exec %W(pg_dump -U #{database_user} --clean --no-owner --no-privileges #{database_name})
-      File.open(fname, 'w') {|f| f.write(res.out.to_s) }
-      fname
+      unless is_sqlite
+        fname = "#{working_dir}/database-#{Time.now.to_i}-#{rand 100000}.sql"
+        $stderr.puts "dumping database to #{fname}"
+        res = @state_container.exec %W(pg_dump -U #{database_user} --clean --no-owner --no-privileges #{database_name})
+        File.open(fname, 'w') {|f| f.write(res.out.to_s) }
+        fname
+      else 
+        # TODO [mlokhova] dump sqlite db somewhere outside of the container
```

Comment:
remove this "TODO" and log instead "skipping database dump (sqlite)"

Reply:
just realized that this was addressed in the next commit - squash commits to avoid churn within the same PR